### PR TITLE
BUGFIX: SD-861 SD-876 update supported properties and remove handle center/zoom on AD

### PR DESCRIPTION
**Pull request by lvkhanh**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [x] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [x] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [x] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [x] Successful `extended test - examples`
- [x] Successful `extended test - storybook`
- [x] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer: https://github.com/gooddata/gdc-analytical-designer/pull/2879

# Related Jira tasks
https://jira.intgdc.com/browse/SD-861
https://jira.intgdc.com/browse/SD-876

### CODE REVIEW

**Comment by lvkhanh** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6489/ : SUCCESS in 14m 02s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2282/ : SUCCESS in 8s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3369/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2283/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/2770/ : SUCCESS in 18m 13s

**Comment by lvkhanh** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1441.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2284/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/2771/ : SUCCESS in 32m 36s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6505/ : SUCCESS in 9m 16s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2304/ : SUCCESS in 5s

File: src/internal/components/configurationPanels/GeoPushpinConfigurationPanel.tsx [hunk 1]
```diff
@@ -6,7 +6,8 @@ import UnsupportedProperties from "../configurationControls/UnsupportedPropertie
 export default class GeoPushpinConfigurationPanel extends ConfigurationPanelContent {
     public componentDidMount() {
         this.props.pushData({
-            references: null,
+            properties: this.props.properties,
+            references: this.props.references,
             ignoreUndoRedo: true,
         });
```
Comment (ptquang86) [inline src/internal/components/configurationPanels/GeoPushpinConfigurationPanel.tsx]: :)
`const { properties, pushData , references } = this.props;`

File: src/internal/components/configurationPanels/tests/GeoPushpinConfigurationPanel.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import * as React from "react";
+import { shallow } from "enzyme";
+import { IConfigurationPanelContentProps } from "../ConfigurationPanelContent";
+import { DEFAULT_LOCALE } from "../../../../constants/localization";
+import { VisualizationTypes } from "../../../../constants/visualizationTypes";
+import GeoPushpinConfigurationPanel from "../GeoPushpinConfigurationPanel";
+import UnsupportedProperties from "../../configurationControls/UnsupportedProperties";
+
+describe("GeoPushpinConfigurationPanel", () => {
+    function createComponent(props: IConfigurationPanelContentProps) {
```
Comment (ptquang86) [inline src/internal/components/configurationPanels/tests/GeoPushpinConfigurationPanel.spec.tsx]: please add the return type for this function. 
It should be `ShallowWrapper`

File: src/internal/components/configurationPanels/tests/GeoPushpinConfigurationPanel.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        return shallow<IConfigurationPanelContentProps, null>(<GeoPushpinConfigurationPanel {...props} />, {
+            lifecycleExperimental: true,
```
Comment (ptquang86) [inline src/internal/components/configurationPanels/tests/GeoPushpinConfigurationPanel.spec.tsx]: I can't find this option in enzyme docs :(

File: src/internal/components/configurationPanels/tests/GeoPushpinConfigurationPanel.spec.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        featureFlags: {
+            enablePushpinGeoChart: true,
+        },
+        isError: false,
+        isLoading: false,
+        locale: DEFAULT_LOCALE,
+        type: VisualizationTypes.PUSHPIN,
+    };
+
+    it("should unsupported configuration properties", () => {
```
Comment (ptquang86) [inline src/internal/components/configurationPanels/tests/GeoPushpinConfigurationPanel.spec.tsx]: we need a verb in this sentence :)

File: src/internal/components/configurationPanels/tests/GeoPushpinConfigurationPanel.spec.tsx (hunk 3 above)
Comment (naibinh) [reply src/internal/components/configurationPanels/tests/GeoPushpinConfigurationPanel.spec.tsx]: @lvkhanh , please help me double check, removing { lifecycleExperimental: true } does not affect test result. It still goes green.

File: src/internal/components/configurationPanels/tests/GeoPushpinConfigurationPanel.spec.tsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+        const wrapper = createComponent({
+            ...defaultProps,
+            references: {
+                a779214c00d24bdb9071c4b2ff6b57ef: "/gdc/md/pid/obj/88",
```
Comment (naibinh) [inline src/internal/components/configurationPanels/tests/GeoPushpinConfigurationPanel.spec.tsx]: `a779214c00d24bdb9071c4b2ff6b57ef ` could be `tooltip_text_reference`, mock data should be simplelized IMO :)

Comment (lvkhanh) [reply src/internal/components/configurationPanels/tests/GeoPushpinConfigurationPanel.spec.tsx]: updated

File: src/internal/components/configurationPanels/tests/GeoPushpinConfigurationPanel.spec.tsx (hunk 4 above)
Comment (lvkhanh) [reply src/internal/components/configurationPanels/tests/GeoPushpinConfigurationPanel.spec.tsx]: updated

File: src/internal/components/configurationPanels/tests/GeoPushpinConfigurationPanel.spec.tsx (hunk 3 above)
Comment (lvkhanh) [reply src/internal/components/configurationPanels/tests/GeoPushpinConfigurationPanel.spec.tsx]: updated

File: src/internal/components/configurationPanels/tests/GeoPushpinConfigurationPanel.spec.tsx (hunk 2 above)
Comment (lvkhanh) [reply src/internal/components/configurationPanels/tests/GeoPushpinConfigurationPanel.spec.tsx]: updated

File: src/internal/components/configurationPanels/GeoPushpinConfigurationPanel.tsx (hunk 1 above)
Comment (lvkhanh) [reply src/internal/components/configurationPanels/GeoPushpinConfigurationPanel.tsx]: updated

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by ptquang86** [comment]
LGTM

**Comment by naibinh** [comment]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3381/ : SUCCESS in 19m 36s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6516/ : SUCCESS in 10m 57s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1568/ : SUCCESS in 14s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2256/ : SUCCESS in 24m 25s

**Pull request merged**